src/view/uniprotkb/components/FreeTextView.tsx

```diff
@@ -1,43 +1,39 @@
 import React, { Fragment } from 'react';
 import { v1 } from 'uuid';
 import UniProtEvidenceTag from '../../../components/UniProtEvidenceTag';
-import { FreeText } from '../../../model/types/CommentTypes';
+import { FreeText, TextWithEvidence } from '../../../model/types/CommentTypes';
 
 type FreeTextProps = {
   comments?: FreeText[];
-  includeTitle?: boolean;
+  title?: string;
 };
 
-const FreeTextView: React.FC<FreeTextProps> = ({
+export const TextView: React.FC<{ comments: TextWithEvidence[] }> = ({
   comments,
-  includeTitle = false,
-}) => {
+}) => (
+  <section className="text-block" key={v1()}>
+    {comments.map(comment => (
+      <Fragment key={v1()}>
+        {comment.value}
+        {comment.evidences && (
+          <UniProtEvidenceTag evidences={comment.evidences} />
+        )}
+      </Fragment>
+    ))}
+  </section>
+);
+
+const FreeTextView: React.FC<FreeTextProps> = ({ comments, title }) => {
   if (!comments || comments.length <= 0) {
     return null;
   }
-  const freeTextData = comments.map(item => (
-    <span className="text-block" key={v1()}>
-      {item.texts &&
-        item.texts.map(itemText => {
-          return (
-            <Fragment key={v1()}>
-              {itemText.value}
-              {itemText.evidences && (
-                <UniProtEvidenceTag evidences={itemText.evidences} />
-              )}
-            </Fragment>
-          );
-        })}
-    </span>
-  ));
+  const freeTextData = comments.map(
+    item => item.texts && <TextView comments={item.texts} key={v1()} />
+  );
 
   return (
     <Fragment>
-      {includeTitle && (
-        <h4 style={{ textTransform: 'capitalize' }}>
-          {comments[0].commentType.toLowerCase()}
-        </h4>
-      )}
+      {title && <h4 style={{ textTransform: 'capitalize' }}>{title}</h4>}
```

Comment:
We have so far not used put styles inline like this - is this intentional?

Reply:
Nope! thanks for the heads up